index.js

```diff
@@ -60,8 +62,8 @@ interface Answer {
   qCode: String
   label: String
   type: AnswerType!
-  mandatory: Boolean
```

Comment:
Can you keep all `mandatory` fields in place for backwards compatibility (whilst the schema is merged but API/front-end/publisher are not). You can mark as deprecated with the `@deprecated` directive:

```gql
mandatory: Boolean @deprecated(reason: "Use `properties` instead.")
```

Reply:
Good idea. Just updated PR.